src/js/index.js

```diff
@@ -0,0 +1,48 @@
+const capitalize = string => string[0].toUpperCase() + string.substr(1, string.length);
+
+const reverseString = string => string
+  .split('')
+  .reverse()
+  .join('');
+
+const getChar = (char) => {
+  let charCode = char.charCodeAt(0) + 3;
```

Comment:
caesar should be passed the rotation key, it shouldn't be hardcoded. 
currently, your implementation seems to use 3, and for my test Caesar cipher doesn't work at all